### DISCUSSION TITLE
Please help me to resolve this issue

### DISCUSSION THREAD

**Discussion by priyankahdp** [ccbogel/QualCoder#888] [Q&A] [answered]
I've followed all steps [Documentation](https://github.com/ccbogel/QualCoder/wiki/01-Introduction) and tried to install on Ubuntu 20.04. Everything went well until this command.

`python3 -m pip install .`
when I gave this command it returned me an error.

```
(qualcoder) user@linux:~/qualcoder$ python3 -m pip install .
ERROR: Directory '.' is not installable. Neither 'setup.py' nor 'pyproject.toml' found.
```

Please help me to resolve this issue.
Thanks & Regards
Priyan K.

**Comment by ccbogel** [comment]
OK. I am presuming you downloaded the latest code with the green Code zip download button.

Are you in the QualCoder-master folder? (As this line **~/qualcoder$**  tells me that you are probably not). 

e.g. for me on Windows:
(env) C:/Users/cc/Downloads/**QualCoder-master**>

The QualCoder-master folder contains the setup.py script.

So try this above recommendation first. And let me know.

Secondly, while not related to the above problem, you are using an older Ubuntu OS version, so I am not sure how well it would work (e.g. potentially other errors). But you can let me know.

**Comment by priyankahdp** [comment]
Thanks for your prompt response. Herewith I attached my folder location and available files in there.
Also that error which I've faced
-Thanks again
![qualcoder_installation_error](https://github.com/ccbogel/QualCoder/assets/2636903/b1f58a2a-ef21-48b7-a587-09d4100e39bd)

**Reply by ccbogel** [reply, marked as answer]
No this is not right. You need to be in the QualCoder folder to run the commands for the env and install:
![image](https://github.com/ccbogel/QualCoder/assets/5834638/cb8fcd07-5ef7-40ce-8c19-fddd64d33ad8)

Then:
![image](https://github.com/ccbogel/QualCoder/assets/5834638/f621431e-2d8a-4967-9ac5-9ab1a3074df0)

Then:
![image](https://github.com/ccbogel/QualCoder/assets/5834638/6b6397a6-4bb6-4248-b2e9-38030eeb2d47)

Then to run 

`python3 -m qualcoder`

or 
`qualcoder`

**Reply by priyankahdp** [reply]
Yes exactly, I was going to type it now
Thank you so much, now I'm able to start application
finally I got xcb error but it resolved using this link
[https://forum.qt.io/topic/93247/qt-qpa-plugin-could-not-load-the-qt-platform-plugin-xcb-in-even-though-it-was-found/99](https://forum.qt.io/topic/93247/qt-qpa-plugin-could-not-load-the-qt-platform-plugin-xcb-in-even-though-it-was-found/99)

Thanks again, Good luck team.!
